docs/src/3.3.Command.mdx

```diff
@@ -2,14 +2,22 @@ import CommandDemo from './jsx/api/Command.js';
 
 # Command
 
-Command is the base class for all regl-based drawing commands. Subclasses should override getDrawProps and (and optionally getHitmapProps). Note: getHitmapProps is not implemented in this base class.
+Command is the base class for all regl-based drawing commands. Wrap this class inside another class to use it.
 
-| Name                        | optional | type                                                                   | Default | Description                                                                                         |
-| --------------------------- | -------- | ---------------------------------------------------------------------- | ------- | --------------------------------------------------------------------------------------------------- |
-| `layerIndex`                | yes      | `number`                                                               |         | defines the draw order of each command, e.g. -1 will put the shape in lower layer                   |
-| `hitmapProps`               | yes      | `T & ({ colors: Vec4[] } | { color: Vec4 })`                           |         | hitmap props built on top of children prop with additional `colors` or `color` field                |
-| `getObjectFromHitmapId`     | yes      | `(objectId: number, hitmapProps: HitmapProps<T>[]) => ?HitmapProps<T>` |         | find the `hitmapProp` from the `hitmapProps` based on the `objectId` generated from the mouse event |
-| `getHitmapId` (deprecating) | yes      | `(T) => ?number`                                                       |         | map the marker object to a `hitmapId`                                                               |
+##### Properties generally implemented by wrapping classes
+
+| Name          | optional | type                        | Default | Description                                                                 |
+| ------------- | -------- | --------------------------- | ------- | --------------------------------------------------------------------------- |
+| `reglCommand` | no       | `(Regl) => (Props) => void` |         | The command that takes the `regl` function and children and draws the shape |
+
+##### Properties generally passed through wrapping classes
+
+| Name         | optional | type               | Default | Description                                                                       |
+| ------------ | -------- | ------------------ | ------- | --------------------------------------------------------------------------------- |
+| `children`   | no       | `Array<any> | any` |         | Arbitrary properties passed into reglCommand.                                     |
```

Comment:
The `Array<any> | any` bit doesn't get rendered properly by the Github markdown render, because it sees the `|` as a cell divider. Just make sure that in our docs this actually looks correct.